dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -108,20 +115,41 @@ private static String osDetectedArch() {
     }
   }
 
-  private static DependencyNode resolveCompileTimeDependencies(
-      DependencyNode rootDependencyArtifact)
+  public DependencyGraphBuilder() {
+    this(ImmutableList.of(CENTRAL.getUrl()));
+  }
+
+  /**
+   * @param mavenRepositoryUrls Maven repository URLs to search for dependencies
+   * @throws IllegalArgumentException if a URL is malformed or not having an allowed scheme
+   */
+  public DependencyGraphBuilder(Iterable<String> mavenRepositoryUrls) {
+    ImmutableList.Builder<RemoteRepository> repositoryListBuilder = ImmutableList.builder();
+    for (String mavenRepositoryUrl : mavenRepositoryUrls) {
+      RemoteRepository repository = mavenRepositoryFromUrl(mavenRepositoryUrl);
+      repositoryListBuilder.add(repository);
+    }
+    this.repositories = repositoryListBuilder.build();
+  }
+
+  /** Returns unresolved artifact problems encountered during constructing a dependency graph. */
```

Comment:
during constructing a --> while constructing the

Reply:
Fixed in DependencyGraphResult.getArtifactProblems.